CHANGES.rst

```diff
@@ -1,6 +1,256 @@
 Traits CHANGELOG
 ================
 
+Release 6.1.0
+-------------
+
+Released: 2020-05-XX
+
+The Traits library is a foundational component of the Enthought Tool Suite. It
+provides observable, typed attributes for Python classes, making those classes
+suitable for event-driven dataflow programming and for immediate use as models
+for graphical user interfaces, like those provided by the TraitsUI library.
+
+Traits 6.1 is the latest feature release in the Traits 6 series, and contains
+several major improvements.
+
+Highlights of this release
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* A new :mod:`observation <traits.observation>` framework for observing traited
+  attributes and other observable objects has been introduced. This is intended
+  to provide a full replacement for the existing :func:`on_trait_change`
+  mechanism, and aims to fix a number of fundamental flaws in the design of
+  that mechanism. See the :ref:`observation <observe-notification>` section of
```

Comment:
This reference is currently broken, but that should be fixed by the open PR #1060

Reply:
Now working.